### ISSUE TITLE
Html5QrcodeScanner error -  Property 'supportedScanTypes' is missing, but required

### ISSUE THREAD

**Issue by ezrabutler** [mebjas/html5-qrcode#530]
Using the sample "Easy Mode" code in Svelte, in VSCode, I received an error of 

```
Argument of type '{ fps: number; qrbox: { width: number; height: number; }; }' is not assignable to parameter of type 'Html5QrcodeScannerConfig'.
  Property 'supportedScanTypes' is missing in type '{ fps: number; qrbox: { width: number; height: number; }; }' but required in type 'Html5QrcodeScannerConfig'.
``` 
and it is not compiling.

 - OS: MacOS Monterey 12.3.1
 - VSCode:  Version: 1.70.1 (Universal))
 - Version: 2.2.1

**Comment by TheSoldier1941** [comment]
Hello! I've had the same problem, but I managed to solve it thanks to seeing the data types required by Visual Studio Code's helper. I'm currently using the library in some Ionic/Angular project with TypeScript.

You must change the arguments passed to the object constructor.
I've noticed you can't use the default values that are shown in the Github documentation, instead, try using something like:
```ts
    const html5QrcodeScanner = new Html5QrcodeScanner(
    'reader',
    { fps: 10, supportedScanTypes: [Html5QrcodeScanType.SCAN_TYPE_CAMERA] },
    /* verbose= */ false)
```
Here I've used Html5QrcodeScanType.SCAN_TYPE_CAMERA, but you're also allowed to use Html5QrcodeScanType.SCAN_TYPE_FILE as well.

I hope this helps! 🦆

**Comment by vincentckk** [comment]
> Hello! I've had the same problem, but I managed to solve it thanks to seeing the data types required by Visual Studio Code's helper. I'm currently using the library in some Ionic/Angular project with TypeScript.
> 
> You must change the arguments passed to the object constructor. I've noticed you can't use the default values that are shown in the Github documentation, instead, try using something like:
> 
> ```ts
>     const html5QrcodeScanner = new Html5QrcodeScanner(
>     'reader',
>     { fps: 10, supportedScanTypes: [Html5QrcodeScanType.SCAN_TYPE_CAMERA] },
>     /* verbose= */ false)
> ```
> 
> Here I've used Html5QrcodeScanType.SCAN_TYPE_CAMERA, but you're also allowed to use Html5QrcodeScanType.SCAN_TYPE_FILE as well.
> 
> I hope this helps! 🦆

The latest version is okay.

**Comment by mebjas** [comment]
Fixed in v2.2.4

**Comment by mebjas** [comment]
What error are you seeing?

**Comment by expdts** [comment]
@mebjas
```const html5QrcodeScanner = new Html5QrcodeScanner('qReader', {fps: 10, qrbox: {width: 250, height: 250}}, false);```

``
Argument of type '{ fps: number; qrbox: { width: number; height: number; }; }' is not assignable to parameter of type 'Html5QrcodeScannerConfig'.   Property 'supportedScanTypes' is missing in type '{ fps: number; qrbox: { width: number; height: number; }; }' but required in type 'Html5QrcodeScannerConfig'
``

**Comment by mebjas** [comment]
Fixed in v2.3.2 and published.

Please validate.
